.github/pull_request_template.md

```diff
@@ -1,49 +1,56 @@
-<!--
-  Your description goes here: A summary of the change, including any relevant motivation and context.
+# What it does
 
-  If relevant, include a description both of the existing behavior and of the new behavior.
+<!--
+  A summary of the change, including any relevant background, motivation, and context.
+  If relevant, include a description, screenshots, and/or video of the existing and new behavior.
 -->
 
+# Links
+
 <!--
-  Other aspects to consider. uncomment and add detail for any that seem necessary:
+  Links to relevant external resources; ie, specification documents, Jira tickets, related PRs, Honeybadger errors, etc.
 -->
 
-<!-- ### Background -->
-<!-- ### Privacy -->
-<!--
-1.	Does the Project involve the collection, use or sharing of new Personal Data?
+- [spec]()
+- [Jira ticket]()
```

Comment:
yes, that bothers me too! good idea, i'll update

Reply:
actually, on second thought, i think i'll remove these bullets entirely. this will leave the "Links" header with a comment explaining what it should contain, which i think is enough here

Reply:
We could just put these in a comment; the original idea was to just give an example of some links you may commonly want to include

Reply:
is it sufficient that the comment describes those common links? or do you think it would be useful/easier to have a separate comment below that one similar to dave's original suggestion:

```
<!--
- spec: []()
- jira ticket: []
-->
```

Reply:
@Hamms i'm not sure what thumbs-up means in this context 😅  do you prefer the former or the latter option?

Reply:
Haha, I can see how that was unclear. I don't have a strong opinion either way, but I lean toward the latter option; I like the idea of having a mini-template in a comment that you can use to jumpstart your "links" section if you so desire

Reply:
understandable -- added!